src/test/java/uk/gov/hmcts/ccd/domain/model/common/DisplayContextParameterUtilTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterUtil.updateCollectionDisplayContextParameter;
+
+class DisplayContextParameterUtilTest {
+
+    @Test
+    @DisplayName("should create collection parameter)")
+    void shouldCreateCollectionParameter() {
+        assertAll(
+            () -> assertThat(updateCollectionDisplayContextParameter("", asList("allowInsert", "allowDelete")),
+                is("#COLLECTION(allowDelete,allowInsert)")),
+            () -> assertThat(updateCollectionDisplayContextParameter("", asList("allowInsert")),
+                is("#COLLECTION(allowInsert)")),
+            () -> assertThat(updateCollectionDisplayContextParameter("", asList("allowDelete")),
+                is("#COLLECTION(allowDelete)")),
+            () -> assertThat(updateCollectionDisplayContextParameter("", asList("")),
+                is("#COLLECTION()"))
+                 );
+    }
+
+    @Test
+    @DisplayName("should update parameter)")
+    void shouldUpdateParameter() {
+        assertAll(
+            () -> assertThat(updateCollectionDisplayContextParameter("#COLLECTION(allowInsert,allowDelete)", asList("allowInsert", "allowDelete")),
+                is("#COLLECTION(allowDelete,allowInsert)")),
```

Comment:
this one is funny, how come it's inverting the order?

Reply:
Because we always order the results.

Reply:
👍